python/unblob/report.py

```diff
@@ -275,9 +282,27 @@ def log_with(self, logger):
         logger.warning(self.log_msg, path=self.path)
 
 
-@attrs.define(kw_only=True, frozen=True)
-class PathTraversalProblem(ExtractionProblem):
+class ExtractionProblem(ExtractionProblemBase):
+    """A non-fatal problem discovered during extraction.
+
+    A report like this still means, that the extraction was successful,
+    but there were problems that got resolved.
+    The output is expected to be complete, with the exception of
+    the reported path.
+
+    Examples
+    --------
+    - duplicate entries for certain archive formats (tar, zip)
+    - unsafe symlinks pointing outside of extraction directory
+
+    """
+
+    report_type: Literal["ExtractionProblem"] = "ExtractionProblem"
+
+
+class PathTraversalProblem(ExtractionProblemBase):
     extraction_path: str
+    report_type: Literal["PathTraversalProblem"] = "PathTraversalProblem"
```

Comment:
You probably need to adapt typing for `record_problem` in `python/unblob/file_utils`:

```
  /home/runner/work/unblob/unblob/python/unblob/file_utils.py:518:33 - error: Argument of type "PathTraversalProblem" cannot be assigned to parameter "problem" of type "ExtractionProblem" in function "record_problem"
    "PathTraversalProblem" is not assignable to "ExtractionProblem" (reportArgumentType)
```